install.sh

```diff
@@ -86,7 +91,7 @@ function uninstall_legacy() {
 
 function uninstall() {
     if [ "$SHOULD_PRE_UNINSTALL" = true ]; then
-        ./pre-uninstall.sh
+        ./pre-uninstall.sh $( (( NO_SUDO == true )) && echo "--no-sudo" )
```

Comment:
for safety, should be replaced by `"$([ "$NO_SUDO" == true ] && echo "--no-sudo")"`